logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/usesCases/GetOngoingCallsUseCase.kt

```diff
@@ -1,5 +1,7 @@
-package com.wire.kalium.logic.feature.call
+package com.wire.kalium.logic.feature.call.usesCases
```

Comment:
Singular, non-camelCased package naming?

```suggestion
package com.wire.kalium.logic.feature.call.usescase
```